test/tests/api/v2_0/obm_settings.py

```diff
@@ -0,0 +1,101 @@
+from config.api2_0_config import *
+from on_http_api2_0 import ApiApi as Api
+from on_http_api2_0 import rest
+from modules.logger import Log
+from json import dumps, loads
+
+LOG = Log(__name__)
+
+"""
+Class to abstract the RackHD Out-of-Band settings 
+"""
+class obmSettings(object):
```

Comment:
If there aren't a lot of data structure differences here, it might make more sense to move obm_settings.py to /modules and pass in the API object to the constructor?